src/types/sequence.rs

```diff
@@ -330,6 +389,9 @@ impl<'v> PyTryFrom<'v> for PySequence {
 
 #[cfg(test)]
 mod test {
+    macro_rules! create_tests {
```

Comment:
Here not all tests are for `FromPyObject`.
Do we need to repeat all of them?
I think we really need it array version of `test_extract_bytearray_to_vec`.